ui/src/app/certificate_requests/row.tsx

```diff
@@ -47,6 +48,26 @@ export default function Row({ id, csr, certificate, ActionMenuExpanded, setActio
             warningText: "Deleting a Certificate Request means this row will be completely removed from the application. This action cannot be undone."
         })
     }
+    const handleCopy = () => {
+        navigator.clipboard.writeText(csr).then(function () {
+            setSuccessNotification("CSR copied to clipboard")
+            setTimeout(() => {
+                setSuccessNotification(null);
+            }, 2500);
+        }, function (err) {
+            console.error('could not copy text: ', err);
+        });
+    }
+    const handleDownload = () => {
+        const blob = new Blob([csr], { type: 'text/plain' });
+        const link = document.createElement('a');
+        link.href = URL.createObjectURL(blob);
+        link.download = "csr.pem"; // TODO: change this to <csr-commonname>.pem
```

Comment:
We should probably suffix this with the CSR ID (or with the common name as mentioned in this todo). When we download more than one they all have the same file name

Reply:
I'll append ID to the name of the file, and keep the todo as is, since it's waiting on #31